src/libraries/System.Private.CoreLib/src/System/Reflection/Emit/Opcode.cs

```diff
@@ -45,7 +45,15 @@ internal OpCode(OpCodeValues value, int flags)
         internal bool EndsUncondJmpBlk() =>
             (m_flags & EndsUncondJmpBlkFlag) != 0;
 
-        internal int StackChange() =>
+        /// <summary>
+        /// The value of how the IL instruction changes the evaluation stack.
+        /// </summary>
+        /// <remarks>
+        /// The difference between how many elements are popped from the stack and how many are pushed onto the stack as a result of the IL instruction.
+        /// For some IL instructions like <see cref="OpCodes.Call"/> this value is not fixed and depends on the called reference signature, for example if it is
+        /// calling a method, depends on the method signature (push count depends on the returning a value, pop count depends how many parameters passed).
```

Comment:
How is this look?
```suggestion
        /// For some IL instructions like <see cref="OpCodes.Call"/> this value is not fixed and depends on the called reference signature. 
        /// In this case you should not rely on this value for calculating stack size and/or max stack, instead need to evaluate the reference signature.
        /// For example, in case it is calling a method, need to evaluate the method signature, the push count depends on the returning value,
        /// the pop count depends on how many parameters passed.
```

Reply:
In the first sentence, "this value" means "The difference between how many elements are popped from the stack and how many are pushed onto the stack".

In the second sentence, "this value" means "the value returned by this property".

Avoid saying "this value" for clarity?

Also, I think we should document what the function returns when the value is not fixed.